src/encoded/audit/file.py

```diff
@@ -148,6 +149,14 @@ def audit_paired_with(value, system):
     if 'paired_with' not in value:
         return
 
+    if value['file_format'] == 'fastq' and value['paired_with'].get('file_format') != 'fastq':
```

Comment:
It's almost always better to use the get function ```value.get('file_format')``` over the key value method ```value['file_format']```, since the latter will crash when the key does not exist (DNE).  Get will automatically return None if it DNE. 

One exception would be if we already know it exists, like with the paired_with key in value dict.   We return early if IT doesn't exist so we don't need to check afterward in the conditional. 

Also we're getting the file_format paired_with value twice, may as well declare it as a variable.  Makes the whole block easier to read. 
```
paired_with_file_format = value['paired_with'].get('file_format')
if value.get('file_format') == 'fastq' and paired_with_file_format != 'fastq':
   ...etc
```
Then use paired_with_file_format in the detail message too.

Reply:
DONE